### PR TITLE
update azure_rm_availabilityset related module document

### PR DESCRIPTION
**Pull request by Fred-sun**
##### SUMMARY

Update azure_rm_availabilityset related module document.

##### ISSUE TYPE

- Docs Pull Request

##### COMPONENT NAME

azure_rm_availabilityset/azure_rm_availabilityset_facts
##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

File: lib/ansible/modules/cloud/azure/azure_rm_availabilityset.py [hunk 1]
```diff
@@ -18,11 +18,12 @@
 
 version_added: "2.4"
 
-short_description: Manage Azure availability set.
+short_description: Manage Azure availability set
 
 description:
-    - Create, update and delete Azure availability set. An availability set cannot be updated, you will have to
-      recreate one instead. The only update operation will be for the tags.
+    - Create, update and delete Azure availability set.
```
Comment (yuwzho) [inline lib/ansible/modules/cloud/azure/azure_rm_availabilityset.py]: "Azure Availability Set"

File: lib/ansible/modules/cloud/azure/azure_rm_availabilityset.py [hunk 2]
```diff
@@ -46,11 +47,11 @@
             - Valid azure location. Defaults to location of the resource group.
     platform_update_domain_count:
         description:
-            - Update domains indicate groups of virtual machines and underlying physical hardware that can be rebooted at the same time. Default is 5.
+            - Update domains indicate groups of virtual machines and underlying physical hardware that can be rebooted at the same time. Default is C(5).
```
Comment (yuwzho) [inline lib/ansible/modules/cloud/azure/azure_rm_availabilityset.py]: "Default is C(5)." can be removed

File: lib/ansible/modules/cloud/azure/azure_rm_availabilityset.py [hunk 3]
```diff
@@ -46,11 +47,11 @@
...
         default: 5
```
Comment (yuwzho) [inline lib/ansible/modules/cloud/azure/azure_rm_availabilityset.py:52]: type: int

File: lib/ansible/modules/cloud/azure/azure_rm_availabilityset.py [hunk 4]
```diff
@@ -93,7 +94,7 @@
 
 RETURN = '''
 state:
-    description: Current state of the availability set
+    description: Current state of the availability set.
     returned: always
     type: dict
```
Comment (yuwzho) [inline lib/ansible/modules/cloud/azure/azure_rm_availabilityset.py]: fill the contains, the options are mentioned in https://github.com/ansible/ansible/blob/254e9c62d411daed37c1f1b5c564a0db75e91a34/lib/ansible/modules/cloud/azure/azure_rm_availabilityset.py#L115

File: lib/ansible/modules/cloud/azure/azure_rm_availabilityset.py [hunk 5]
```diff
@@ -46,11 +47,11 @@
             - Valid azure location. Defaults to location of the resource group.
     platform_update_domain_count:
         description:
-            - Update domains indicate groups of virtual machines and underlying physical hardware that can be rebooted at the same time. Default is 5.
+            - Update domains indicate groups of virtual machines and underlying physical hardware that can be rebooted at the same time. Default is C(5).
         default: 5
     platform_fault_domain_count:
         description:
-            - Fault domains define the group of virtual machines that share a common power source and network switch. Should be between 1 and 3. Default is 3
+            - Fault domains define the group of virtual machines that share a common power source and network switch. Should be between C(1) and C(3). Default is C(3).
```
Comment (yuwzho) [inline lib/ansible/modules/cloud/azure/azure_rm_availabilityset.py]: Write in new line, and don't need the "Default is 3".
type: int

File: lib/ansible/modules/cloud/azure/azure_rm_availabilityset_facts.py [hunk 6]
```diff
@@ -17,21 +17,21 @@
 
 version_added: "2.4"
 
-short_description: Get availability set facts.
```
Comment (yuwzho) [inline lib/ansible/modules/cloud/azure/azure_rm_availabilityset_facts.py:20]: Return in facts module also need to be updated

**Review by yuwzho — CHANGES_REQUESTED** [review]

File: lib/ansible/modules/cloud/azure/azure_rm_availabilityset.py [hunk 7]
```diff
@@ -93,9 +97,19 @@
 
 RETURN = '''
 state:
-    description: Current state of the availability set
+    description: Current state of the availability set.
     returned: always
     type: dict
+    example: {
```
Comment (yuwzho) [inline lib/ansible/modules/cloud/azure/azure_rm_availabilityset.py]: It should not only give an example. But some yaml files, you can refer the servicebus module’s doc

**Review by yuwzho — CHANGES_REQUESTED** [review]

**Comment by Fred-sun** [comment]
ready_for_review

**Review by zikalino — APPROVED** [review]

**Comment by yuwzho** [comment]
@Fred-sun Please also fill the return part of the facts module

**Comment by yuwzho** [comment]
shipit

**Comment by yungezz** [comment]
shipit

**Comment by acozine** [comment]
Merging based on the double shipits. Thanks everybody.

**Pull request merged**